capsules/src/net/udp/driver.rs

```diff
@@ -0,0 +1,494 @@
+//! UDP userspace interface for transmit and receive.
+//!
+//! Implements a userspace interface for sending and receiving UDP messages.
+//! Also exposes a list of interface addresses to the application (currently
+//! hard-coded.
```

Comment:
This comment should describe the high level operation of the driver (e.g. "Processes use this driver to send UDP packets from a common interface and bind to UDP ports for receiving packets." etc).

Reply:
updated